db/system_distributed_keyspace.cc

```diff
@@ -335,17 +337,33 @@ static set_type_impl::native_type prepare_cdc_streams(const std::vector<cdc::str
     return ret;
 }
 
+// cdc_streams_descriptions mutation
```

Comment:
Maybe name the function get_cdc_streams_descriptions_mutation instead of adding a comment?